src/Libraries/DSIronPython/IronPythonEvaluator.cs

```diff
@@ -54,7 +55,7 @@ public class IronPythonEvaluator : Dynamo.PythonServices.PythonEngine
         /// </summary>
         public const string packageExtraFolderName = @"extra";
 
-        public override string Name => PythonNodeModels.PythonEngineVersion.IronPython2.ToString();
+        public override string Name => PythonEngineManager.IronPython2EngineName;
```

Comment:
@pinzart90 do you need to make changes to the published ironPython package as well?

Reply:
yes. after this PR